txscript/standard.go

```diff
@@ -350,6 +359,17 @@ func PayToAddrScript(addr btcutil.Address) ([]byte, error) {
 	return nil, ErrUnsupportedAddress
 }
 
+// NullDataScript creates a provably prunable script
+// containing OP_RETURN followed by the passed data.
+func NullDataScript(data []byte) ([]byte, error) {
+	if len(data) > MaxDataCarrierSize {
+		return nil, ErrStackLongScript
+	}
+
+	builder := NewScriptBuilder()
+	return builder.AddOp(OP_RETURN).AddData(data).Script()
```

Comment:
I know I originally showed this on two lines, but I forgot it was in the `txscript` package itself, so it'll fit on a single line.

Mind making it the following single line?

```Go
return NewScriptBuilder().AddOp(OP_RETURN).AddData(data).Script()
```